src/api-reference/common/ess/topics/expense/v4.expense-events.md

```diff
@@ -3,117 +3,172 @@ title: Concur Expense Event
 layout: reference
 ---
 
-{% include prerelease.html %}
```

Comment:
@visadin Please return the pre-release notice.

Reply:
updated with the pre-release tag